deploy/vagrant/conf/gce.yml.template

```diff
@@ -0,0 +1,69 @@
+# Setting up your Google Cloud Project
+#
+# The Project ID for your Google Cloud Platform account.
+Google_Cloud_Project_ID:
+
+# The Client Email address for your Service Account.
+Service_Acount_Email_Address:
+
+# The location of the JSON private key file matching your Service Account.
+json_Key_Location: 
+
+
+#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
+#
+# This is the path to SSH keys. The easiest way to setup GCE keys is using the 
+# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to 
+# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
+# directly or manually manage keys.
+Key_Path: "~/.ssh/google_compute_engine"
```

Comment:
hmmm. Not sure what to think of that. The AWS setup defaults to ~/.ssh as
well. But if it worked for you, I'm tempted to use an absolute path.
Except, as the path contains a username, I either couldn't have a default
value or make an assumption that it is the same as the SSH_USERNAME.

On Tue, Mar 22, 2016 at 4:36 PM Chaomin <notifications@github.com> wrote:

> In deploy/vagrant/conf/gce.yml.template
> <https://github.com/Alluxio/alluxio/pull/2889#discussion_r57088494>:
>
> > +Google_Cloud_Project_ID:
> > +
> > +# The Client Email address for your Service Account.
> > +Service_Acount_Email_Address:
> > +
> > +# The location of the JSON private key file matching your Service Account.
> > +json_Key_Location:
> > +
> > +
> > +#SSH - Vagrant/Ansible will SSH into VMs for environment setup.
> > +#
> > +# This is the path to SSH keys. The easiest way to setup GCE keys is using the
> > +# CLI (gcloud compute config-ssh) Which creates metatdata that is provided to
> > +# every VM launched. Alternatively, it is possible to add to the GCE metatdata service
> > +# directly or manually manage keys.
> > +Key_Path: "~/.ssh/google_compute_engine"
>
> I ran into a issue using "~/.ssh/google_compute_engine". But switching to
> absolute path, "/Users/chaomin/.ssh/google_compute_engine" would work. Is
> this related with vagrant parsing path? Would you suggest using absolute
> path here?
>
> === Logs====
> {"Google_Cloud_Project_ID"=>"corded-guild-125817",
> "Service_Acount_Email_Address"=>"chaomin@alluxio.com",
> "json_Key_Location"=>"~/.ssh/Project-12e448f09873.json",
> "Key_Path"=>"~/.ssh/google_compute_engine", "SSH_Username"=>"chaomin",
> "Preemptible"=>false, "Machine_Type"=>"n1-standard-1", "Scopes"=>["
> https://www.googleapis.com/auth/cloud-platform"], "Disk_Size"=>10,
> "Tags"=>"Alluxio", "Name"=>nil, "Network"=>"default",
> "Image"=>"centos-7-v20160301"}
> There are errors in the configuration of this machine. Please fix
> the following errors and try again:
>
> Google Provider:
> * Private key for Google could not be found in the specified location.
>
> —
> You are receiving this because you were mentioned.
> Reply to this email directly or view it on GitHub
> <https://github.com/Alluxio/alluxio/pull/2889/files/10c44f950b47d9fd4f9523368b0ed683524d9e74#r57088494>
>